jobs/tox.ini

```diff
@@ -5,9 +5,13 @@ skip_missing_interpreters = True
 [testenv]
 usedevelop=true
 commands =
+    python setup.py develop
     py.test --cov-report term-missing --cov buildhub tests {posargs}
+
 deps =
-    -rdev-requirements.txt
+    -r{toxinidir}/requirements/dev.txt
+    -r{toxinidir}/requirements/default.txt
+    -c{toxinidir}/requirements/constraints.txt
 
 install_command = pip install --pre {opts} {packages}
```

Comment:
I don't think we need the `--pre` any more since we're not allowing installation of anything else except exactly what we're prepared to install.

Reply:
Yes indeed